<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>818</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>华文琥珀</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background:#DCDCDC;
border-right:1px solid black;
border-left:1px solid black;
border-top:1px solid black;
border-bottom:1px solid black;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}
QPushButton{
    border:none;
    border-bottom:1px solid black;
    font-size:18px;
    font-weight:700;
 }
QPushButton:hover{border-left:4px solid grey;font-weight:700;}
QCheckBox{
border:none}
</string>
  </property>
  <widget class="QGroupBox" name="faceRecognizerSettingsLogGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>661</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>人脸识别设置</string>
   </property>
   <widget class="QCheckBox" name="equalizeHistCheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>直方图均衡化</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>391</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{border:none}</string>
    </property>
    <property name="title">
     <string>Debug设置</string>
    </property>
    <widget class="QCheckBox" name="debugCheckBox">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>92</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>调试模</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="confidenceThresholdLabel">
        <property name="font">
         <font>
          <family>华文中宋</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{border:none;}</string>
        </property>
        <property name="text">
         <string>置信度阈值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="confidenceThresholdSlider">
        <property name="maximum">
         <number>80</number>
        </property>
        <property name="pageStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>181</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="autoAlarmThresholdLabel">
        <property name="font">
         <font>
          <family>华文中宋</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{border:none;}</string>
        </property>
        <property name="text">
         <string>自动报警阈值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="autoAlarmThresholdSlider">
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>80</number>
        </property>
        <property name="pageStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>65</number>
        </property>
        <property name="sliderPosition">
         <number>65</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="confidencelabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{border:none}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="alarmlabel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{border:none}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="debugLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{border:none;}</string>
    </property>
    <property name="text">
     <string>注：以上调整只有开启调试模式才会触发</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="realTimeCaptureGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>661</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>摄像头实时画面</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="Capturelabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>摄像头未开启</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="logGroup">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>460</y>
     <width>281</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>系统日志</string>
   </property>
   <widget class="QTextEdit" name="logtextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>261</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Voicerecognition">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>60</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>-1</pointsize>
     <weight>87</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>语音识别</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="functionGroupBox">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>180</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>功能开关</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>211</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="faceTrackerCheckBox">
       <property name="font">
        <font>
         <family>华文中宋</family>
        </font>
       </property>
       <property name="text">
        <string>人脸跟踪</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="faceRecognizerCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>华文中宋</family>
        </font>
       </property>
       <property name="text">
        <string>人脸识别</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="panalarmCheckBox">
       <property name="font">
        <font>
         <family>华文中宋</family>
        </font>
       </property>
       <property name="text">
        <string>报警系统</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bellCheckBox">
       <property name="font">
        <font>
         <family>华文中宋</family>
        </font>
       </property>
       <property name="text">
        <string>设备报警</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="left_close">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>20</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="left_visit">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>20</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="left_mini">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>20</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Openvideo">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>摄像头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Voicevideo">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>120</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>-1</pointsize>
     <weight>87</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>语音播报</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
